cmake/versions.cmake

```diff
@@ -0,0 +1,29 @@
+###############################################################################
+#
+# (c) Copyright IBM Corp. 2017
+#
+#  This program and the accompanying materials are made available
+#  under the terms of the Eclipse Public License v1.0 and
+#  Apache License v2.0 which accompanies this distribution.
+#
+#      The Eclipse Public License is available at
+#      http://www.eclipse.org/legal/epl-v10.html
+#
+#      The Apache License v2.0 is available at
+#      http://www.opensource.org/licenses/apache2.0.php
+#
+# Contributors:
+#    Multiple authors (IBM Corp.) - initial implementation and documentation
+###############################################################################
+
+
+#TODO figure out method of sharing version info with makefile build
+set(OMR_VERSION_MAJOR 0)
+set(OMR_VERSION_MINOR 0)
+set(OMR_VERSION_PATCH 1)
+set(OMR_VERSION ${OMR_VERSION_MAJOR}.${OMR_VERSION_MINOR}.${OMR_VERSION_PATCH})
+
+
+set(OMR_VERSION_STRING "<Unknown>" CACHE STRING "") # TODO: Set the version
+set(OMR_JIT_VERSION_STRING "<Unknown>" CACHE STRING "") # TODO: Set the version
```

Comment:
I'm not a big fan of TODOs in general.  In this context they're fairly obvious TODOs aren't they?